### PR TITLE
Add link button as a reference to the command

### PR DESCRIPTION
**Pull request by qhuynh96**
* Ref #6485
* Click on the link button automatically copy the command url to clipboard.
* Screenshot:

screenshot from 2018-01-28 15-10-18

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
I think it's pretty common for these to go on the left margin? WDYT?

File: prow/cmd/deck/static/plugin-help-script.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ function redrawOptions() {
 
 window.onload = function () {
     // set dropdown based on options from query string
+    const hash = window.location.hash;
```
Comment (stevekuznetsov) [inline prow/cmd/deck/static/plugin-help-script.js]: If someone uses this link in a doc, or pastes it to someone, then we turn on more plugins and the page gets re-rendered, what happens to the viability of the link?

Comment (qhuynh96) [reply prow/cmd/deck/static/plugin-help-script.js]: as long as the id of the command does not change, it can be reachable

**Comment by BenTheElder** [comment]
It definitely is for headings at least, but the link is probably the least
important part for users looking through the table so it seems better to
have the star icon etc. on the left instead?
I'm not sure it matters all that much.

File: prow/cmd/deck/static/plugin-help-script.js [hunk 2]
```diff
@@ -361,3 +420,17 @@ function redraw() {
...
+ * Extracts a command name from a command example.
+ * @param {string} commandExample
+ * @return {string}
+ */
+function extractCommandName(commandExample) {
+    const command = commandExample.split(" ");
+    if (!command || command.length === 0) {
+        throw new Error("Cannot extract command name.");
+    }
+    return command[0].slice(1).split("-").join("_");
```
Comment (BenTheElder) [inline prow/cmd/deck/static/plugin-help-script.js]: *I* know what this does, but maybe document what this is actually doing? It's not going to be immediately obvious why this does slice etc.

Comment (qhuynh96) [reply prow/cmd/deck/static/plugin-help-script.js]: done

**Comment by BenTheElder** [comment]
This LGTM, we can further tweak in a follow-up if there are any futher bugs / complaints. I want to start linking people to commands soon.

**Pull request merged**